### PR TITLE
Fix to Putting in Description for Launch Template Resource

### LINKED ISSUE #7510: Launch Template Not Putting in Description
_This issue was originally opened by @brianpham as hashicorp/terraform#20297. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Terraform Version
```
Terraform v0.11.11
+ provider.aws v1.57.0
```

### Terraform Configuration Files
```hcl
resource "aws_launch_template" "template" {
  name = "template"
  description = "2019-02-11"
```

Output when running terraform apply that shows the description in there

### Debug Output

```
module.region_ap_southeast_2.module.launch_template.aws_launch_template.webrtc-template: Creating...
  block_device_mappings.1.ebs.#:                   "" => "1"
  block_device_mappings.1.ebs.0.iops:              "" => "<computed>"
  block_device_mappings.1.ebs.0.volume_size:       "" => "20"
  block_device_mappings.1.ebs.0.volume_type:       "" => "gp2"
  default_version:                                 "" => "<computed>"
  description:                                     "" => "2018-11-06"
```

### Expected Behavior

```
Launch Template Description Field on the AWS Console should show "2019-02-11"
```

### Actual Behavior

```
Launch Template Description is showing up Not specified
```

### Steps to Reproduce

```
1. Create simple terraform launch template resource 
2. Have something in the description
3. Run terraform plan and it will show you that the description is correct
4. Run terraform apply and check AWS console to see if there is anything in the description
```

### PR DESCRIPTION
**Pull request by kterada0509**
Fixes #7510

Changes proposed in this pull request:

* Fixed parameters to putting description
* Add default value for description

Output from acceptance testing:

```basic
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSLaunchTemplate_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -parallel 20 -run=TestAccAWSLaunchTemplate_basic -timeout 120m
=== RUN   TestAccAWSLaunchTemplate_basic
=== PAUSE TestAccAWSLaunchTemplate_basic
=== CONT  TestAccAWSLaunchTemplate_basic
--- PASS: TestAccAWSLaunchTemplate_basic (27.60s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	27.682s
```

```description
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSLaunchTemplate_description'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -parallel 20 -run=TestAccAWSLaunchTemplate_description -timeout 120m
=== RUN   TestAccAWSLaunchTemplate_description
=== PAUSE TestAccAWSLaunchTemplate_description
=== CONT  TestAccAWSLaunchTemplate_description
--- PASS: TestAccAWSLaunchTemplate_description (43.27s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	43.367s
```

### CODE REVIEW

File: aws/resource_aws_launch_template.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ func resourceAwsLaunchTemplate() *schema.Resource {
 			"description": {
 				Type:         schema.TypeString,
 				Optional:     true,
+				Default:      "Managed by Terraform",
```
Comment (bflad) [inline aws/resource_aws_launch_template.go]: We should skip adding our own default here. 👍 Its an old practice and will cause unexpected differences for existing Terraform configurations when added like this.

Comment (kterada0509) [reply aws/resource_aws_launch_template.go]: @bflad Thanks. I removed default description

**Comment by kterada0509** [comment]
```
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSLaunchTemplate_description'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -parallel 20 -run=TestAccAWSLaunchTemplate_description -timeout 120m
=== RUN   TestAccAWSLaunchTemplate_description
=== PAUSE TestAccAWSLaunchTemplate_description
=== CONT  TestAccAWSLaunchTemplate_description
--- PASS: TestAccAWSLaunchTemplate_description (43.44s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	43.551s
```

**Comment by bflad** [comment]
This has been released in [version 1.60.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1600-february-22-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Pull request merged**